RoboJackets/robocup-software
# Merge field comp to ros2

## Description
Removes the special hardcoded-IP field-comp branch we used to have and replaces it with config files and flags. Now our field comp can run sim plays and on real robots without a lot of manual labor.

## Steps to Test
### Test Case 1
1. Get on field comp
2. Run make run-real
3. Run make run-sim
4. Run make run-manual
5. Run make run-real/sim-ex

**Expected result:** all should work as intended

## Review Checklist
